site/content/docs/user/loadbalancer.md

```diff
@@ -24,32 +24,35 @@ description: |-
 
 ---
 
-## Installing metallb using default manifests
+## Installing MetalLB using default manifests
```

Comment:
I think this is a breaking change to the heading id for incoming links, but that seems fine this time